### ISSUE TITLE
regaining focus in the media browser when it is already open

### ISSUE THREAD

**Issue by ranetto** [jcsteh/osara#1072]
Good morning, when I am in the media browser and I am scrolling through the elements, with enter for example, I insert one into a reaper track. In this situation reaper regains focus and the media browser remains open but we cannot act on it and we are forced to close and reopen it to work with it again. In this case escape doesn't work either, because clearly you can't use the media browser as it isn't focused. I ask if it was possible to have a dare action that brings the focus back to the mediabrowser window which is already open but which does not have the focus. This would allow us to not have to press control alt and the letter X to hide it and then the same combination again to reopen it and therefore consequently still get the focus on it. Do you think it's possible? Thank you!

**Comment by ScottChesworth** [comment]
Strange, focus doesn't move from Media Explorer here. It's not the first time I've heard of this happening though. Must be some local setting? Can anyone else reproduce?

**Comment by pitermach** [comment]
Yes, I can reproduce this here, specifically this seems to happen when you insert something on a new track, which also causes reaper to make that track selected and that moves focus out of the media explorer. This personally never bothered me too much and I just got used to pressing the key again to return to the media explorer, but to each their own ☺️

**Comment by ranetto** [comment]
Exactly it happens to me too in that case, I too have always pressed the combination twice, but being able to have a dare action for this it would only be a matter of pressing that combination once and above all by not having to close and reopen our screenreader it would remain well focused on the point in the media list where he was, in fact, often losing focus he refocuses on the tree or something else.
